cypress.config.ts

```diff
@@ -1,39 +1,38 @@
-import { defineConfig } from "cypress";
-const {
-  addMatchImageSnapshotPlugin,
-} = require('@simonsmith/cypress-image-snapshot/plugin');
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { defineConfig } from 'cypress';
+const { addMatchImageSnapshotPlugin } = require('@hyperkid/cypress-image-snapshot/plugin');
```

Comment:
Any reason for custom cypress image? I'd be more comfortable with forking it to RHInsights repository and deploying it under our org.

Reply:
Yeah, it has locked peer dependency of cypress... I will open a PR in the original package but don't want to wait. I also don't want to take ownership of the package. I see this only as a temporary swap.

Reply:
@karelhala fix was published in https://github.com/simonsmith/cypress-image-snapshot/pull/4. I've used the original dependency.